src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -158,13 +172,13 @@ const ChartsHolder = (props: {
             </MainContentInner>
             <ClaimStateBlock
               stateId={props.stateId}
-              countyId={props.countyId}
+              countyId={props.county.county_url_name}
```

Comment:
Looks like the state pages are broken. 😬  https://covid-projections-git-county-share.covidactnow.now.sh/us/wa

`props.county?.county_url_name` might fix it?

Reply:
Updated!